lib/chef_backup/helpers.rb

```diff
@@ -82,6 +82,7 @@ def ensure_file!(file, exception, message)
 
   def shell_out(*command)
     cmd = Mixlib::ShellOut.new(*command)
+    cmd.timeout ||= config['shell_out_timeout'] unless config['shell_out_timeout'].nil?
```

Comment:
I don't think you want `||=` here.  `cmd.timeout` is going to return the default timeout so this could will never set the timeout.